.github/workflows/release-version.yaml

```diff
@@ -0,0 +1,42 @@
+name: Create versioned release
+
+permissions:
+  contents: write
+  packages: write
+
+on:
+  workflow_dispatch:
+    inputs:
+      update-type:
+        type: choice
+        description: Which version you want to increment? Use major, minor or patch
+        required: true
+        default: patch
+        options:
+          - major
+          - minor
+          - patch
+      label:
+        description: Add Labels. i.e final, alpha, rc
+        required: false
+      pre-release:
+        type: boolean
+        description: Pre-release
+        required: false
+        default: false
+
+jobs:
+  release-version:
+    name: Create SemVer releases
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Create version releases
+        uses: climpr/semver-release@v0
```

Comment:
Should be @v1

Reply:
proceeding with v0 as agreed. v1 will come after testing succeeds